src/Workers/Worker.ts

```diff
@@ -1 +1,59 @@
-// nothing yet...
\ No newline at end of file
+import cv from '../../build/opencv_js'
```

Comment:
continuing on the `opencv` property, here we import opencv so it's not necessary to have an `opencv` property.